src/app/component/header/header.component.scss

```diff
@@ -34,4 +34,9 @@ button:hover{
 }
```

Comment:
On line 29 why whole button is applied ? Please add class and please change for span tags too

Reply:
on line 4 why container is used directly ? boostrap class ? please remove that too

Reply:
I have Remove container